datadump/txtparse.c

```diff
@@ -11,7 +11,7 @@ char * parse_buffer (const unsigned char * buffer, unsigned buffer_length) {
   for (pos = 0; pos < buffer_length; pos ++)
     switch (shift_state) {
       case 0:
-        if ((buffer[pos] & 0xfe) == 0xfc)
+        if (buffer[pos] == 0xF7 || buffer[pos] == 0xF8 || buffer[pos] == 0xF9 || buffer[pos] == 0xFC || buffer[pos] == 0xFD)
```

Comment:
Use lowercase for hex constants (for consistency)

Reply:
Actually, might be better to change the logic here, and instead test whether the string in text_table is empty i.e. first character is NUL (which it should be for these control codes).

Reply:
0xf7, 0xf8 and 0xf9 all have defined strings. Are they wrong?